### PR TITLE
Added Contributor guide

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/oah-bes-vm/issues/27

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 7.0 or latest
+   * [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) 2.14.4 or latest
+   * [Git](https://git-scm.com/downloads)
+   * Vim editor: `sudo apt install vim`   
+- ### Install oah-shell
+      curl -L https://raw.githubusercontent.com/Be-secure/oah-installer/master/install.sh | bash
+- ### Update oah-init.sh 
+  - open `oah-init.sh` using vim editor.
+    ```bash
+    sudo vim $HOME/.oah/bin/oah-init.sh
```
Comment (asa1997) [inline CONTRIBUTING.md:19]: @pramit-d Is these lines necessary?

Exporting them from the terminal would be enough to change the namespace.
`export OAH_NAMESPACE=${OAH_NAMESPACE:=Be-Secure}` this line means that, if `OAH_NAMESPACE` is empty set it to `Be-Secure`.

So if we ask the user to export them manually, can't we avoid the steps where he edits the init file.

Moreover, theres a possibility that the user might push it along with the changes.

**Review by asa1997 — COMMENTED** [review]
@pramit-d Please address the comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pramit-d) [reply CONTRIBUTING.md:19]: @asa1997 Yes, we can skip these steps. User can export the namespace from the terminal without modifying the oah-init.sh file.

Comment (asa1997) [reply CONTRIBUTING.md]: @pramit-d Please do confirm the same.

Comment (pramit-d) [reply CONTRIBUTING.md]: Done

**Review by asa1997 — APPROVED** [review]
@pramit-d LGTM

**Review by sudhirverma — APPROVED** [review]

**Pull request merged**